data-transfer-mysql-mmy.tf

```
@@ -1,33 +1,44 @@
-# Infrastructure for the Yandex Cloud Managed Service for MySQL cluster and Data Transfer.
+# Infrastructure for the Yandex Cloud Managed Service for MySQL cluster and Data Transfer
 #
 # RU: https://cloud.yandex.ru/docs/managed-mysql/tutorials/data-migration
 # EN: https://cloud.yandex.com/en/docs/managed-mysql/tutorials/data-migration
 #
-# Set source and target clusters settings.
+# Set source and target clusters settings
 locals {
-  zone_a_v4_cidr_blocks = "10.1.0.0/16" # Set the CIDR block for subnet in the ru-central1-a availability zone.
   # Source cluster settings:
-  source_user    = ""   # Set the source cluster username.
-  source_db_name = ""   # Set the source cluster database name.
-  source_pwd     = ""   # Set the source cluster password.
-  source_host    = ""   # Set the source cluster master host IP address or FQDN.
-  source_port    = 3306 # Set the source cluster port number that Data Transfer will use for connections.
+  source_user    = ""   # Set the source cluster username
+  source_db_name = ""   # Set the source cluster database name
+  source_pwd     = ""   # Set the source cluster password
+  source_host    = ""   # Set the source cluster master host IP address or FQDN
```

```suggestion
  source_host    = ""   # IP address or FQDN of the master host in the source cluster
```
Слишком длинная цепочка сущ-ных, ничего непонятно

---

Исправила.